src/views/docs-view/loaders/remote-content.ts

```diff
@@ -251,6 +251,7 @@ export default class RemoteContentLoader implements DataLoader {
 			// Remove item if it's an empty path (for non-latest versions, the version is prepended to the path)
 			if (
 				navData.navData.length > 1 &&
+				navData.navData[0].path &&
 				navData.navData[0].path.split('/').length > 1
```

Comment:
We should be able to use [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) here to get the same result

```suggestion
				navData.navData[0].path?.split('/').length > 1
```